src/components/price/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { cart } from '@wix/ecom';
+import { products } from '@wix/stores';
+import { PriceInfo } from './price';
+
+export function getCartItemPriceInfo(cartItem: cart.LineItem | undefined): PriceInfo {
+    return {
+        fullPrice: cartItem?.fullPrice?.formattedConvertedAmount,
+        price: cartItem?.price?.formattedConvertedAmount,
+    };
+}
+
+export function getProductPriceInfo(priceData: products.PriceData | undefined): PriceInfo {
```

Comment:
this is redundant util.. as well as the one aboce. Let's just read directly from the data object where it's needed.

Keep in mind that it's a template project, and we should strive to as simple code as possible.

Reply:
removed, but I thought it makes sense to have some utils to simplify usage